.github/workflows/build.yml

```diff
@@ -49,7 +49,7 @@ jobs:
           git-config-email: github-actions[bot]@users.noreply.github.com
 
       - name: Post gh pages link under Checks
-        if: always()
+        if: github.ref != 'refs/heads/main'
```

Comment:
But it won't post if build fails on main? Or it really needs PR number or something similar for that to work?

Reply:
it will
![image](https://user-images.githubusercontent.com/9084735/155727005-54864ee7-a9e2-4317-8e51-28529b7a0b30.png)